main.nf

```diff
@@ -571,6 +572,8 @@ if(params.aligner == 'star'){
             --readFilesCommand zcat \\
             --runDirPerm All_RWX \\
             --outFileNamePrefix $prefix $seqCenter \\
```

Comment:
I think it's this backslash that was causing the problem...
```suggestion
             --outFileNamePrefix $prefix $seqCenter
```

Reply:
Ok lets see - indexing never ran, and my notebook locally doesn't allow STAR testing due to memory issues ...